modules/migration-using-cpma.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+// migration/migrating-openshift-3-to-4.adoc
+[id='migration-using-cpma_{context}']
+= Using CPMA
+
+.Prerequisites
+
+The following prerequisites apply to the {product-title} 3 cluster:
```

Comment:
I don't think it's in the template to include a sentence like this. I would just be specific in each of your prereqs that it it applies to the 3.x cluster.